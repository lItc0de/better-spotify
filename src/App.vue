<template>
  <x-app>
    <layout id="app">
      <router-view/>
      <!-- <div>
        <player v-if="$route.name !== 'Login'"/>
      </div> -->
    </layout>
  </x-app>
</template>

<script>
import Layout from '@/components/Layout.vue';
// import Player from '@/containers/Player.vue';

export default {
  components: {
    Layout,
  },

  props: {
    source: String,
  },
};
</script>
