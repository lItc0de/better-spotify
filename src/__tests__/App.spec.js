import { shallowMount, createLocalVue } from '@vue/test-utils';
import VueRouter from 'vue-router';
import App from '@/App.vue';

const localVue = createLocalVue();
localVue.use(VueRouter);
const router = new VueRouter();

let wrapper;

describe('App', () => {
  beforeEach(() => {
    wrapper = shallowMount(App, {
      localVue,
      router,
      stubs: {
        'x-app': true,
      },
    });
  });

  it('matches the snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
