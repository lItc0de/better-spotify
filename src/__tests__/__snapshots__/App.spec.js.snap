// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App matches the snapshot 1`] = `
<div id="app">
  <div id="nav">
    <vuecomponent-stub to="[object Object]"></vuecomponent-stub> |
    <vuecomponent-stub to="[object Object]"></vuecomponent-stub>
  </div>
  <vuecomponent-stub></vuecomponent-stub>
</div>
`;
